dataflowengineoss/src/main/scala/io/joern/dataflowengineoss/queryengine/SourcesToStartingPoints.scala

```diff
@@ -194,7 +194,7 @@ abstract class BaseSourceToStartingPoints extends Callable[Unit] {
         val usageInput = targetsToClassIdentifierPair(literalToInitializedMembers(lit), src)
         val uses       = usages(usageInput)
         val globals = globalFromLiteral(lit, recursive = false).flatMap {
-          case x: Identifier if x.isModuleVariable => x :: moduleVariableToFirstUsagesAcrossProgram(x)
+          case x: Identifier if x.isModuleVariable => moduleVariableToFirstUsagesAcrossProgram(x)
```

Comment:
My reasoning for removing `x` here was: if `x` is a global/module-level variable, but it isn't used anywhere according to `moduleVariableToFirstUsageAcrossProgram`, does it make sense to keep it? Keeping it introduces some false-positives in some recently-added Python dataflow tests, but its exclusion doesn't change anything (as far I as could see by locally testing)